ch2-git-basics/cloning.qmd

```diff
@@ -0,0 +1,57 @@
+## Cloning an Existing Repository
+
+### Introduction
+
+This section will guide you through the process of cloning a GitHub repository. Cloning a repository means making a local copy of a project that exists on GitHub. This enables you to make changes and test locally before pushing to be approved and merged with the main branch on GitHub.
+
+There are several ways to clone an existing repository. There are two methods detailed below. The first uses Rstudio and the second uses Git Bash. Start with step 1, which is the same for both, and then decide which method to follow. Once you have successfully cloned, you should see the files appear in both the file you have chosen to store your repository in and the files pane in R studio.
+
+### Step 1: Clone the Repository
+
+a\. Go to the [git-academy-sandbox](https://github.com/dfe-analytical-services/git-academy-sandbox) repository page on GitHub.
+
+b\. Click the "Code" green button and copy the URL (HTTPS link).
+
+### To Clone Using RStudio:
+
+#### Step 2a: Open RStudio
+
+Open RStudio on your computer.
+
+#### Step 2b: Clone the Repository in RStudio
+
+-   In RStudio, go to **File \> New Project \> Version Control \> Git**.
+
+-   Paste the URL you copied from GitHub into the "Repository URL" field.
+
+-   Choose a directory on your computer where you want to save the repository.
+
+-   Click "Create Project".
+
+#### Step 2c: Verify the Clone
+
+Check the Files pane in RStudio to see if the repository files have been cloned to your local machine. The files pane normally opens in the right-hand bottom corner of RStudio.
+
+### Or, to Clone Using Git Bash:
+
+### Step 2a: Open folder and Git Bash
+
+Navigate to the folder where you want to clone the repository. Right-click in the directory and select "Git Bash Here".
+
+#### Step 2b: Clone the Repository in Git Bash
+
+-   In the Git Bash window that opens, use the \`git clone\` command followed by the URL you copied from GitHub. For this repository, we would write:
+
+**git clone**
+
+**https://github.com/dfe-analytical-services/git-academy-sandbox.git**
+
+You will need to right-click to paste, as the keyboard shortcut will not work in Git Bash.
+
+#### Step 2c: Verify the Clone
+
+Check the Files pane in RStudio to see if the repository files have been cloned to your local machine. The files pane normally opens in the right-hand bottom corner of RStudio.
```

Comment:
I think for this one we should be IDE agnostic. Analysts could be working in R, Python, SQL, html+css, etc, so for the git bash one, I think we shouldn't use R-Studio at all.

Instructions for checking the repo in git bash would be along the lines of:

- Navigate into the repo directory, e.g. `cd git-academy-sandbox`
- You should now see the current branch name (e.g. main) at the end of the filepath
- type `git log` to view recent commits or `git status` to check the state of the cloned repository (at this point it should say everything's up to date as you haven't made any changes to anything yet)